socrates/lib/eventstore/RegistrationReadModel.js

```diff
@@ -14,10 +14,12 @@ function RegistrationReadModel(eventStore, soCraTesReadModel) {
   this._soCraTesReadModel = soCraTesReadModel;
```

Comment:
Das ReadModel brauchen wir nur noch [hier](https://github.com/softwerkskammer/Agora/pull/1080/files#diff-1ba0124995009e83d9e58ecb4b4ab184R173), ich würde das in die Methode reingeben.